app/_locales/en/messages.json

```diff
@@ -3195,6 +3192,9 @@
   "noSnaps": {
     "message": "You don't have any snaps installed."
   },
+  "noThanks": {
```

Comment:
For the other locales, we have a "No thanks" already in the key `onboardingMetametricsDisagree`. Perhaps we can rename that key to `noThanks` and use it in both places? That would let us ship this quickly without waiting for new translations, we could use what we have already.